### PR TITLE
Personal Item: Michael's Card

### PR DESCRIPTION
**Pull request by AemonTillado**
## About the PR
This PR adds a personal item for Michael Hands - a Father's Day card gifted to him by his son.

## Why / Balance
Allows Michael to spawn with this sentimental item which he currently prints from a fax machine at the start of every shift.

## Technical details
The card is parented from in-game books, and the sprite is based on the default cover_base book sprite. It has been altered to be edible in the same manner of paper, only with double the waiting time and providing three fiber instead of one (to represent consuming an entire cardboard rectangle as opposed to a single sheet of paper!)

## Media
card1
card2
card3
card4
card5

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Personal item for Michael Hands - Michael's Card.

### CODE REVIEW

**Comment by Coolsurf6** [comment]
: )

File: Resources/Prototypes/_Umbra/Entities/Objects/Misc/Books/michaels_card.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    flavors:
+    - paper
+  - type: BadFood
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 3
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 3
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Objects/Misc/Books/michaels_card.yml:32]: Remove the SolutionContainerManager component, this should not be modified from the base item.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
One issue needs to be addressed (caused by personal items rules). I am deciding to allow the extended eating delay for the reason provided by the PR author.

GM approval when review is addressed.

File: Resources/Prototypes/_Umbra/Entities/Objects/Misc/Books/michaels_card.yml (hunk 1 above)
Comment (TsjipTsjip) [reply Resources/Prototypes/_Umbra/Entities/Objects/Misc/Books/michaels_card.yml:32]: Well that's just SS14 scrunkliness for you isn't it... Fine. Accepted as-is.

**Review by TsjipTsjip — APPROVED** [review]

**Pull request merged**